### PR TITLE
De 1021 personality insights v3

### PR DESCRIPTION
**Pull request by mediumTaj**
### Summary

Personality insights V3 abstraction

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1563,61 +1563,153 @@ The IBM Watson™ [Personality Insights][personality_insights] service enables a
...
...vice available in as little as 18 months.\n\n\"I have been in medical education for 40 years and we're still a very memory-based curriculum,\" said Dr. Herbert Chase, a professor of clinical medicine at Columbia University who is working with I.B.M. on the physician's assistant. \"The power of Watson- like tools will cause us to reconsider what it is we want students to do.\"\n\nI.B.M. executives also said they are in discussions with a major consumer electronics retailer to develop a version of Watson, named after I.B.M.'s founder, Thomas J. Watson, that would be able to interact with consumers on a variety of subjects like buying decisions and technical support.\n\nDr. Ferrucci sees none of the fears that have been expressed by theorists and science fiction writers about the potential of computers to usurp humans.\n\n\"People ask me if this is HAL,\" he said, referring to the computer in \"2001: A Space Odyssey.\" \"HAL's not the focus; the focus is on the computer on 'Star Trek,' where you have this intelligent information seek dialogue, where you can ask follow-up questions and the computer can look at all the evidence and tries to ask follow-up questions. That's very cool.\"";
```
Comment (germanattanasio) [inline README.md]: There should be a way to reduce this text. Maybe a file ? Maybe a placeholder like `<text-here>`

File: Scripts/Services/PersonalityInsights/v3/DataModels.cs [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+		/// </summary>
+		public const string ARABIC = "ar";
+		/// <summary>
+		/// Spanish.
+		/// </summary>
+		public const string SPANISH = "es";
+		/// <summary>
+		/// Japanese
+		/// </summary>
+		public const string JAPANESE = "ja";
```
Comment (germanattanasio) [inline Scripts/Services/PersonalityInsights/v3/DataModels.cs]: I think the API accept 8 different languages

File: Scripts/Services/PersonalityInsights/v3/PersonalityInsights.cs [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+		#endregion
+
+		#region Profile
+		private const string SERVICE_GET_PROFILE = "/v3/profile";
+
+		public delegate void OnGetProfile(Profile profile, string data);
+
+		public bool GetProfile(OnGetProfile callback, string source,
+			string contentType = ContentType.TEXT_PLAIN,
+			string contentLanguage = Language.ENGLISH,
```
Comment (germanattanasio) [inline Scripts/Services/PersonalityInsights/v3/PersonalityInsights.cs]: Make sure that `Language` has all the supported languages. See the java-sdk

File: Scripts/Services/PersonalityInsights/v3/DataModels.cs (hunk 2 above)
Comment (mediumTaj) [inline Scripts/Services/PersonalityInsights/v3/DataModels.cs]: is `Content-Language` in the dropdown not accurate?
https://watson-api-explorer.mybluemix.net/apis/personality-insights-v3#!/personality-insights/profile

File: Examples/ServiceExamples/Scripts/ExamplePersonalityInsightsV3.cs [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+*
+*/
+
+using UnityEngine;
+using IBM.Watson.DeveloperCloud.Services.PersonalityInsights.v3;
+using IBM.Watson.DeveloperCloud.Logging;
+
+public class ExamplePersonalityInsightsV3 : MonoBehaviour
+{
+	PersonalityInsights m_personalityInsights = new PersonalityInsights();
```
Comment (mediumTaj) [inline Examples/ServiceExamples/Scripts/ExamplePersonalityInsightsV3.cs]: Yes, I'm actually doing both. There is string testString and string dataPath. In the Start method we call GetProfile using dataPath then we call GetProfile using the testString. I don't have access to Application.dataPath in the constructor so I need to set it in the Start method. I guess I can define first and set it when we start?

File: README.md (hunk 1 above)
Comment (mediumTaj) [inline README.md]: OK I'll revise

**Pull request merged**